### PR TITLE
[Icons Upgrade] Fix incompatabilities

### PR DESCRIPTION
**Pull request by damassi**
Follow-up to https://github.com/artsy/palette/pull/308 which fixes some stuff around 
- No more state full icons; e.g., HeartIcon -> HeartIcon + HeartFillIcon
- No more unnecessary displayNames; since all icons are now SFC's we get displayNames for free
- No more inconsistent colors; interfaces guard against passing anything not defined in theme, and no need for `color` helper
- No more inconsistent heights / widths / size props; these props all inherit from Icon base component
- Icons now have titles, so users and screen-readers and so on can get context.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit d519c6b3a976ff46a0d86ee753731f2d96510138

https://deploy-preview-2140--artsy-reaction.netlify.com

**Comment by damassi** [comment]
(Merging this so that we can be sure everything is working before folks come in tomorrow! Sorry sorry.)

**Comment by artsyit** [comment]
:rocket: PR was released in v15.0.0 :rocket:

File: src/Components/v2/__tests__/CountdownTimer.test.tsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ describe("CountdownTimer", () => {
 
       const text = timer.text()
       expect(text).toMatchInlineSnapshot(
-        `"01d 00s leftRespond by Dec 04, 8:50 AM ESTExpired offers end the negotiation process permanently."`
+        `"time remaining01d 00s leftRespond by Dec 04, 8:50 AM ESTExpired offers end the negotiation process permanently."`
```
Comment (alloy) [inline src/Components/v2/__tests__/CountdownTimer.test.tsx]: Why did these change?

Comment (damassi) [reply src/Components/v2/__tests__/CountdownTimer.test.tsx]: All of our SVG icons now include a `<title>time remaining<title>` tag. These fields have icons embeddded inline with the text.

**Pull request merged**